connect/runtime/src/main/java/org/apache/kafka/connect/runtime/Worker.java

```diff
@@ -648,6 +648,12 @@ private WorkerTask buildWorkerTask(ClusterConfigState configState,
         // These settings will execute infinite retries on retriable exceptions. They *may* be overridden via configs passed to the worker,
         // but this may compromise the delivery guarantees of Kafka Connect.
         producerProps.put(ProducerConfig.MAX_BLOCK_MS_CONFIG, Long.toString(Long.MAX_VALUE));
+        // By default, producers that are instantiated and used by Connect have idempotence disabled even after idempotence became
+        // default for Kafka producers. This ensures Connect continues to work with many Kafka broker versions, including older brokers that do not support
```

Comment:
Minor nit, but I think it's worth making it super clear that _Connect_ is disabling the idempotent behavior of all it's producers, versus somehow the _producers_ doing something to disable idempotent behavior.
```suggestion
        // By default, Connect disables idempotent behavior for all producers, even though idempotence became
        // default for Kafka producers. This is to ensure Connect continues to work with many Kafka broker versions, including older brokers that do not support
```